src/styles.js

```diff
@@ -1,19 +1,67 @@
 import {createGlobalStyle} from 'styled-components';
 
 export const GlobalStyle = createGlobalStyle`
-	*,
-	*::before,
-	*::after {
-		box-sizing: border-box;
-	}
+/* http://meyerweb.com/eric/tools/css/reset/
```

Comment:
Most of it will Never be in use, you could delete them afterwards maybe to have a cleaner Code